archunit/src/main/java/com/tngtech/archunit/library/GeneralCodingRules.java

```diff
@@ -133,4 +134,21 @@ private GeneralCodingRules() {
     @PublicAPI(usage = ACCESS)
     public static final ArchRule NO_CLASSES_SHOULD_USE_JAVA_UTIL_LOGGING =
             noClasses().should(USE_JAVA_UTIL_LOGGING);
+
+    /**
+     * For information about checking this condition, refer to {@link GeneralCodingRules}.
+     */
+    @PublicAPI(usage = ACCESS)
+    public static final ArchCondition<JavaClass> USE_JODATIME =
+            dependOnClassesThat(resideInAPackage("org.joda.time"))
+                    .as("use JodaTime");
+
+    /**
+     * Modern Java projects use the [java.time] API instead of the JodaTime library
+     * <br>
+     * For information about checking this rule, refer to {@link GeneralCodingRules}.
+     */
+    @PublicAPI(usage = ACCESS)
+    public static final ArchRule NO_CLASSES_SHOULD_USE_JODATIME =
+            noClasses().should(USE_JODATIME);
```

Comment:
I'd actually use:
```
noClasses().should(USE_JODATIME).because("modern Java projects use the [java.time] API instead");
```

Reply:
Yes, that actually is a good idea, I'll add that :wink: